docs/blog/2018-09-24-reach-router/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: How we improved Gatsby's accessibility in v2 by switching to @reach/router
+date: 2018-09-24
+author: Amberley Romo
+tags: ["routing", "v2", "accessibility"]
+---
+
+We recently announced the [second major release of Gatsby](/blog/2018-09-17-gatsby-v2/) 🚀. One change we want to highlight is the switch to using [@reach/router](https://reach.tech/router) to improve the accessibility of routing in Gatsby sites. What is @reach/router, and what are the benefits of undertaking the switch?
```

Comment:
Maybe mention Ryan here? Could be nice for him and also make the mention of his name at the end clearer!

Reply:
His name is mentioned directly in the next paragraph -- I think that's enough?

Reply:
Oh somehow I skimmed over his name! Probably is enough.